### PR TITLE
Add recipe for guppy3

### PR DESCRIPTION
**Pull request by fgypas**
Add recipe for guppy3

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

### CODE REVIEW

**Comment by fgypas** [comment]
@conda-forge/help-python-c

I am not the developer of the tool, but I need this dependency in another recipe in biocontainers (https://github.com/bioconda/bioconda-recipes/pull/28506#partial-pull-merging). Can you please help me with that?

**Comment by fgypas** [comment]
@conda-forge/help-python-c

**Comment by dopplershift** [comment]
The package from bioconda will need to be added to conda-forge first.

**Comment by fgypas** [comment]
@conda-forge/help-python-c This is ready for review

**Comment by fgypas** [comment]
@conda-forge/staged-recipes This is ready for review

**Comment by fgypas** [comment]
@conda-forge-admin, please ping team

File: recipes/guppy3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/guppy3-{{ version }}.tar.gz
+  sha256: 2114c70da3dbf90ca34b550407bac3ecd549a0b89184f74c0ae0d70d16abada2
+
+build:
+  number: 0
+  skip: true   # [py2k]
```
Comment (wolfv) [inline recipes/guppy3/meta.yaml:15]: this isn't really necessary anymore since python 2.7 reached EOL.

**Pull request merged**